ls8/ls8.py

```diff
@@ -3,9 +3,13 @@
 """Main."""
 
 import sys
-from cpu import *
+import os
+from cpu import CPU
 
-cpu = CPU()
+path = os.getcwd()
+print(path)
 
-cpu.load()
-cpu.run()
\ No newline at end of file
+with open('./ls8/examples/mult.ls8') as program:
```

Comment:
You'll want to un-hardcode this and replace `'./ls8/examples/mult.ls8'` with `sys.argv[1]`